Plotter/test/testRDataFrame_Sample.py

```diff
@@ -0,0 +1,367 @@
+###    stacks = { } # { selection : { variable: Stack } }
+###    nhists = 0
+###    for selection in selections:
+###      start2 = time.time(), time.process_time() # wall-clock & CPU time
+###      results = sampleset.getstack(variables,selection,method=method,split=split,parallel=parallel,verb=verb)
+###      print(f">>> test_SampleSet_MultiDraw: Running of {len(results)} results with MultiDraw for one selection took {took(*start2)} with {ROOT.GetThreadPoolSize()} threads")
+###      stacks[selection] = results # add nested dictionary
+###      nhists += sum(len(s.exphists)+(1 if s.datahist else 0) for s in results)
+###    print(f">>> test_SampleSet_MultiDraw: Running of {nhists} results with MultiDraw for all selections took {took(*start)} with {ROOT.GetThreadPoolSize()} threads")
+###    
+###    # PLOT
+###    for selection in selections:
+###      for stack in stacks[selection]:
+###        #stack = stacks[selection][variable].getstack()
+###        stack.draw(ratio=True)
+###        stack.drawlegend()
+###        stack.drawtext("Test SampleSet.getstack MultiDraw",selection.title)
+###        stack.saveas(f"{outdir}/testRDF_stack_$VAR_{selection.filename}{tag}_MultiDraw.png")
+###        stack.close(keep=parallel) # avoid segmentation fault for parallel
+###    
+###  else: # no stack
+###    start  = time.time(), time.process_time() # wall-clock & CPU time
+###    hists  = { } # { selection : { variable: HistSet.exp } }
+###    nhists = 0
+###    for selection in selections:
+###      variables_ = [ ] 
+###      for variable in variables:
+###        if variable.plotfor(selection) and selection.plotfor(variable):
+###          variable.changecontext(selection.selection)
+###          variables_.append(variable)
+###      start2 = time.time(), time.process_time() # wall-clock & CPU time
+###      results = sampleset.gethists(variables_,selection,split=split,parallel=parallel,verb=verb)
+###      print(f">>> test_SampleSet_MultiDraw: Running of {len(results)} results with MultiDraw for one selection took {took(*start2)} with {ROOT.GetThreadPoolSize()} threads")
+###      hists[selection] = { v: results[v].exp for v in results} # add nested dictionary
+###      nhists += sum(len(results[v].exp) for v in results)
+###    print(f">>> test_SampleSet_MultiDraw: Running of {nhists} results with MultiDraw for all selections took {took(*start)} with {ROOT.GetThreadPoolSize()} threads")
+###    
+###    # PLOT
+###    for selection in selections:
+###      for variable in hists[selection]:
+###        variable.changecontext(selection.selection)
+###        plot = Plot(variable,hists[selection][variable]) # create Plot object
+###        plot.draw(ratio=True)
+###        plot.drawlegend()
+###        plot.drawtext("Test SampleSet.gethists MultiDraw",selection.title)
+###        plot.saveas(f"{outdir}/testRDF_plot_{variable.filename}_{selection.filename}{tag}_MultiDraw.png")
+###        plot.close()
+
```

Comment:
Can this be removed or is it useful for checks?

Reply:
I kept it in case we ever need to to comparison for debugging.